### PR TITLE
Add fix to generated configuration

### PR DESCRIPTION
**Pull request by wfspotz**
The build.sh script now runs the packages/PyTrilinos/util/configFix.py script, which removes the pytrilinos library from being linked to users' projects.

### CODE REVIEW

**Comment by wfspotz** [comment]
Circle-CI looks good until the very end:

    + test -f /home/circleci/test/build_artefacts/conda-forge-build-done
    + exit 1
    Exited with code 1

File: README.md [hunk 1]
```diff
@@ -14,8 +14,8 @@ Python interfaces to fast, compiled object-oriented algorithms and enabling tech
 Current build status
 ====================
 
-Linux: [![Circle CI](https://circleci.com/gh/conda-forge/pytrilinos-feedstock-conda-forge-feedstock.svg?style=shield)](https://circleci.com/gh/conda-forge/pytrilinos-feedstock-conda-forge-feedstock)
```
Comment (ocefpaf) [inline README.md]: I wonder what caused `pytrilinos-feedstock-conda-forge-feedstock`...

Comment (isuruf) [reply README.md]: That happens when your local git folder's name is different.

Comment (ocefpaf) [reply README.md]: Thanks. I never saw that :grimacing:

**Comment by ocefpaf** [comment]
OK. I restarted CircleCi on master... just in case.
I'll try to build this locally tomorrow to examine why the non-zero exit status.

**Comment by wfspotz** [comment]
@ocefpaf Have you had a chance to examine the non-zero exit status?

**Comment by wfspotz** [comment]
@ocefpaf @isuruf On Oct 24, I plan to give a short talk at the Trilinos Users Group Meeting about the Anaconda/conda-forge installers for Trilinos and PyTrilinos. It would be nice to have this issue worked out before then, if possible.

**Pull request closed without merging**